src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -57,6 +58,16 @@ public void parse_validArgs_returnsFindCommand() {
     }
 
     // Find Descriptor tests
+    @Test
+    public void findDescriptor_validPrefixInput_addToPredicateList() throws ParseException {
+        String validExperiencePrefixInput = " y/3";
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(validExperiencePrefixInput,
+                PREFIX_EXPERIENCE);
+        FindDescriptor findDescriptor = new FindDescriptor(argMultimap);
+        assertTrue(!findDescriptor.getPredicates().isEmpty());
```

Comment:
You can use `assertFalse(findDescriptor.getPredicates().isEmpty())` here instead of `assertTrue(!findDescriptor.getPredicates().isEmpty())` 😸

Reply:
👍